lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/RootContextLauncher.cs

```diff
@@ -71,30 +71,32 @@ internal ContextRuntime GetRootContext()
             return _rootContext;
         }
 
-        private static IInjector InjectServices(Optional<ServiceConfiguration> serviceConfig)
+        private IInjector InjectServices(Optional<IConfiguration> serviceConfig)
         {
             IInjector rootServiceInjector;
 
             if (serviceConfig.IsPresent())
             {
-                rootServiceInjector = TangFactory.GetTang().NewInjector(serviceConfig.Value.TangConfig);
-                InjectedServices services = null;
+                rootServiceInjector = TangFactory.GetTang().NewInjector(serviceConfig.Value);
```

Comment:
Should rootservice injector be forked from evaluator injector after we fix REEF-217? If yes, you might want to add TODO and REEF-217 here as comments.